torch/csrc/autograd/init.cpp

```diff
@@ -503,7 +503,8 @@ static PyObject* is_any_autocast_enabled(PyObject* _unused, PyObject* arg) {
   HANDLE_TH_ERRORS
   if (at::autocast::is_enabled() || at::autocast::is_cpu_enabled() ||
       at::autocast::is_xpu_enabled() || at::autocast::is_ipu_enabled() ||
-      at::autocast::is_xla_enabled() || at::autocast::is_hpu_enabled()) {
+      at::autocast::is_xla_enabled() || at::autocast::is_hpu_enabled() ||
+      at::autocast::is_privateuseone_enabled()) {
```

Comment:
This is already being fixed (along with many other things) in https://github.com/pytorch/pytorch/pull/124359
Are you ok with waiting that other PR is merged to get this and close this PR?

Reply:
ok, thanks!